### PR TITLE
readme: add upstream process and maintainers

### PR DESCRIPTION
**Pull request by xuguangxin**

### CODE REVIEW

**Comment by xuguangxin** [comment]
@uartie could you help add @galinart as collaborator too.

@galinart  please help review this
thanks

**Comment by uartie** [comment]
> @uartie could you help add @galinart as collaborator too.
>
collaborator invite sent.

File: README.cartwheel [hunk 1]
```diff
@@ -4,6 +4,22 @@ Intel developer staging area for upstream patch contributions to [FFmpeg](https:
...
+1. Close PR1, and send it to the community, ping the community with some frequency if it's no response.
+1. If the community has no response in 4 weeks or it’s an urgent issue, send PR2 to https://github.com/intel-media-ci/cartwheel-ffmpeg
+1. Provide upstream patch link, ping maintainers for merging
+1. Ping community until the patch merged in upstream
+
+
+## maintainers
+* @xhaihao for Linux and OneVPL
+* @galinart for Windows
+* @uartie for CI system
```
Comment (uartie) [inline README.cartwheel:21]: Can you add Focus and Bin as backups for CI?

**Comment by galinart** [comment]
> > @uartie could you help add @galinart as collaborator too.
> 
> collaborator invite sent.

Accepted. Thanks

File: README.cartwheel (hunk 1 above)
Comment (xuguangxin) [reply README.cartwheel]: good suggestion added more guys

**Comment by xuguangxin** [comment]
@feiwan1 @guoyejun @Semmer2 @FocusLuo @Bin-CI 
please help review too.
thanks

**Review by FocusLuo — APPROVED** [review]

**Comment by guoyejun** [comment]
looks I'm not the reviewer

**Comment by xuguangxin** [comment]
> looks I'm not the reviewer

this my just at you, not add you to Reviewers. 
May need @uartie help add you to collaborates

File: README.cartwheel [hunk 2]
```diff
@@ -4,6 +4,23 @@ Intel developer staging area for upstream patch contributions to [FFmpeg](https:
 
 The upstream project is tracked as a submodule in this repo.
 
+## upstream process
+To improve upstream patch quality and reduce patchset rebase efforts, we use the following process
+1. Send PR1 to https://github.com/intel-media-ci/ffmpeg, add module maintainers to Reviewers
+1. Got approval from one maintainer
+1. Wait for 1 more workday to see any objections. If it's an urgent issue, please ask another reviewer + 1 for you.
```
Comment (feiwan1) [inline README.cartwheel:11]: Did you intend to use 1/2/3... before each item?

Comment (xuguangxin) [reply README.cartwheel:11]: yes, it's markdown syntax.
see https://github.com/xuguangxin/cartwheel-ffmpeg/tree/process for final result.

Comment (feiwan1) [reply README.cartwheel:11]: OK, I see. Then LGTM.

**Review by uartie — APPROVED** [review]
LGTM

**Pull request merged**